src/google/protobuf/map_field.h

```diff
@@ -837,9 +837,6 @@ class PROTOBUF_EXPORT MapIteratorBase {
   ValueRef value_;
 };
 
-extern template class MapIteratorBase</*kIsMutable=*/false>;
```

Comment:
I don't think we want to delete these lines, because as I understand, these are here to tell the compiler to look for their implementation elsewhere (it's in map_field.cc) instead of generating them again.

Does adding `PROTOBUF_EXPORT` here fix the problem? That is just a guess, but I see that we do something similar [here](https://github.com/protocolbuffers/protobuf/blob/23649b0c8c124b2d3b1fe46752df5107cd2a5322/src/google/protobuf/metadata_lite.h#L214).